assets/i18n/en_US.json

```diff
@@ -129,7 +129,7 @@
         "noneTooltip": "Deselect all patches",
 
         "loadPatchesSelection": "Load patches selection",
-        "noSavedPatches": "No saved patches for the selected app.\nPress Done to save current selection.",
+        "noSavedPatches": "No saved patches for the selected app.\nPress Done to save the current selection.",
```

Comment:
`No saved patch selections for the selected app.\nPress Done to save the current selection.`

Reply:
> `No saved patch selections for the selected app.\nPress Done to save the current selection.`

"selection" shouldn't be plural because this message is for a single app.

I'll change to `No saved patch selection for the selected app.\nPress Done to save the current selection.`